hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/HealthConfiguration.java

```diff
@@ -20,31 +20,41 @@
 
     @Bean
     public SubscriptionHealthProblemIndicator laggingIndicator() {
-        return new LaggingIndicator(healthProperties.getMaxLagInSeconds());
-    }
-
-    @Bean
-    public SubscriptionHealthProblemIndicator slowIndicator() {
-        return new SlowIndicator(healthProperties.getMinSubscriptionToTopicSpeedRatio());
+        if (healthProperties.isLaggingIndicatorEnabled()) {
+            return new LaggingIndicator(healthProperties.getMaxLagInSeconds());
+        }
+        return new DisabledIndicator();
```

Comment:
can we store a single static instance of the `DisabledIndicator` and return everywhere?

Reply:
👍